### PR TITLE
Mmonarch dev

### PR DESCRIPTION
**Pull request by shirubana**

### CODE REVIEW

File: bifacial_radiance/spectral_utils.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        
+        return (lower_bound, upper_bound)
+    
+    def scale_values(self, scaling_factor):
+        self.df['value'] *= scaling_factor
+    
+def spectral_albedo_smarts(zen, azm, material, min_wavelength=300,
+                           max_wavelength=4000):
+    try:
+        from pySMARTS.smarts import SMARTSSpectraZenAzm
```
Comment (kanderso-nrel) [inline bifacial_radiance/spectral_utils.py:141]: @shirubana I was snooping around to see what's going on in bifacial_radiance-land and am interested in this `pySMARTS` package.  I assume it's a python wrapper around SMARTS, but could only find [this](https://github.com/BYU-PRISM/hale-trajectory) after some quick googling.  What's the story on `pySMARTS`?

Comment (shirubana) [reply bifacial_radiance/spectral_utils.py:141]: https://github.com/NREL/py-SMARTS

hmmm... story is, we had a matlab wrapper back at my Univeristy; I got permission from the original grad student, advisor, and made some updates then and talked to Cliff about publishing it in pvlib, but never got around that. Then ported it to python when at NREL, made a pull request on PVLib that I've so far have not done much about and then contacted NREL legal to make sure we were doing things right, which involved having our own Github, copyright notes on all of this situation, and from that github make a PR to pvlib.  I'm just not very good at pvlib whole contributing process and it's been not on my priority (or theirs). But it's there, and it does stuff. I've been meaning to take a fun-work day and do a GUI for my own self, I keep having to go read the manual to understand the inputs and it annoys me. .

Comment (shirubana) [reply bifacial_radiance/spectral_utils.py:141]: also that zen-azimuth function got updated today and I got a better function to work with TMY and SRRL data with a lot more detail. I have to think how to do the inputs mapping more friendly depending on the data inputs available. But working as need hehe.

Comment (kanderso-nrel) [reply bifacial_radiance/spectral_utils.py:141]: Thanks!  I think I was searching google for `"pySMARTS"` without the hyphen, so that's why I missed it.

**Comment by cdeline** [comment]
I'll go ahead and merge this into development.  It still needs some pytests and release notes which we'll have to work up before any new release.

**Pull request merged**